packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/LightCurveViewer/LightCurveViewerContainer.js

```diff
@@ -55,28 +62,28 @@ class LightCurveViewerContainer extends Component {
   }
 
   async requestData () {
+    const { onError } = this.props
     try {
-      this.setState({ loading: asyncStates.loading })
+      // this.setState({ loading: asyncStates.loading })
       const url = this.getSubjectUrl()
       const response = await request.get(url)
-      if (!response.ok) {
-        throw new Error('Invalid response')
-      } else {
+      if (response.ok) {
```

Comment:
If the response isn’t ok, what does this function do?

Reply:
It's an error. Superagent rejects none 200 responses for us, and that error is then caught. I suppose technically the check for `response.ok` doesn't have to happen here because of superagent.